src/astarte-client/client.ts

```diff
@@ -229,16 +229,16 @@ class AstarteClient {
     });
   }
 
-  async getInterfaceNames(): Promise<any> {
+  async getInterfaceNames(): Promise<string[]> {
     const response = await this.$get(this.apiConfig.interfaces(this.config));
     return response.data;
   }
 
-  async getInterfaceMajors(interfaceName: any): Promise<any> {
+  async getInterfaceMajors(interfaceName: string): Promise<number[]> {
     const response = await this.$get(
       this.apiConfig.interfaceMajors({ ...this.config, interfaceName }),
     );
-    return response.data;
+    return response.data.sort();
```

Comment:
adding a sort here changes the behavior of the api. I don't think the client should sort the data, that should be done by the view/screen if they ever need to.